.github/workflows/capsule-cypress.yml

```diff
@@ -6,12 +6,17 @@ on:
       - master
       - develop
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
```

Comment:
question: why are these now needed? Isn't the check on 19 sifficeint?

Reply:
We still need the run to be triggered when the already opened PR switches from draft to ready to review

Reply:
I also believe that the other types are required now that `types` are used, though I couldn't quite tell from github documentation